tools/dotcms-cli/cli/src/main/java/com/dotcms/api/client/files/traversal/task/TraverseParams.java

```diff
@@ -0,0 +1,162 @@
+package com.dotcms.api.client.files.traversal.task;
+
+
+import com.dotcms.api.client.files.traversal.data.Retriever;
+import com.dotcms.cli.common.OutputOptionMixin;
+import java.io.File;
+import java.io.Serializable;
+import org.jboss.logging.Logger;
+
+
+public class TraverseParams implements Serializable {
```

Comment:
Could this not be an Immutable?

Reply:
again TraverseParams exists in the cli module and the immutables dependency is only available in api-data-model